Shared/Samples/Augment reality to navigate route/AugmentRealityToNavigateRouteView.swift

```
@@ -0,0 +1,253 @@
+    @State private var locationDataSource = SystemLocationDataSource()
+    /// A scene with an imagery basemap.
+    @State private var scene = Scene(basemapStyle: .arcGISImagery)
+    /// The elevation surface set to the base surface of the scene.
+    @State private var elevationSurface: Surface = {
+        let elevationSurface = Surface()
+        elevationSurface.navigationConstraint = .unconstrained
+        elevationSurface.opacity = 0.5
+        elevationSurface.backgroundGrid.isVisible = false
+        return elevationSurface
+    }()
+    /// The elevation source with elevation service URL.
+    @State private var elevationSource = ArcGISTiledElevationSource(url: URL(string: "https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer")!)
+    /// The graphics overlay containing a graphic.
+    @State private var graphicsOverlay = GraphicsOverlay()
+    /// The status text displayed to the user.
+    @State private var statusText = "Adjust calibration before starting."
+    /// A Boolean value indicating whether the use is navigatig the route.
+    @State private var isNavigating = false
+    /// The result of the route selected in the route planner view.
+    @State private var routeResult: RouteResult?
+    
+    init() {
+        elevationSurface.addElevationSource(elevationSource)
+        scene.baseSurface = elevationSurface
+    }
+    
+    var body: some View {
+        if isShowingRoutePlanner {
+            RoutePlannerView(isShowing: $isShowingRoutePlanner)
+                .onDidSelectRoute { routeGraphic, routeResult  in
+                    self.routeResult = routeResult
+                    graphicsOverlay = makeRouteOverlay(
+                        routeResult: routeResult,
+                        routeGraphic: routeGraphic
+                    )
+                }
+                .task {
+                    Task { try await elevationSource.load() }
```

`Task` shouldn't be used just to discard an error.

```suggestion
                    try? await elevationSource.load()
```